src/components/dialogs/network-modifications/voltage-init-modification/voltage-init-modification-dialog.tsx

```diff
@@ -97,6 +99,11 @@ interface ShuntCompensatorRowData {
     [TARGET_V]: number | undefined;
 }
 
+interface BusRowData {
+    ID: string;
+    [V]: number | undefined;
```

Comment:
We must also modify the voltage angle of the bus, in addition to his voltage magnitude (see remark in PR : https://github.com/gridsuite/voltage-init-server/pull/57)